front/migrations/db/migration_373.sql

```diff
@@ -0,0 +1,17 @@
+-- Migration created on Oct 06, 2025
+
+-- First add columns as nullable
+ALTER TABLE "webhook_sources_views"
+ADD COLUMN "description" VARCHAR(255),
+ADD COLUMN "icon" VARCHAR(255);
+
+-- Update existing rows with default values
+UPDATE "webhook_sources_views"
+SET "description" = '',
+    "icon" = 'ActionGlobeAltIcon'
+WHERE "description" IS NULL OR "icon" IS NULL;
+
+-- Now make the columns NOT NULL (no default at DB level)
+ALTER TABLE "webhook_sources_views"
+ALTER COLUMN "description" SET NOT NULL,
+ALTER COLUMN "icon" SET NOT NULL;
```

Comment:
Purpose here is to make the column not nullable without having a default value in DB